flyway-database-yugabytedb/src/main/java/org/flywaydb/community/database/postgresql/yugabytedb/YugabyteDBDatabase.java

```diff
@@ -75,4 +70,16 @@ public String getRawCreateScript(Table table, boolean baseline) {
                 "CREATE INDEX IF NOT EXISTS \"" + table.getName() + "_s_idx\" ON " + table + " (\"success\");";
     }
 
-}
\ No newline at end of file
+    @Override
+    public boolean useSingleConnection() {
+        return false;
+    }
+
+    private void init() {
+        try {
+            jdbcTemplate.execute("CREATE TABLE IF NOT EXISTS " + LOCK_TABLE_NAME + " (table_name varchar PRIMARY KEY, locked bool, last_updated timestamp)");
```

Comment:
Put the create table sql string in a constant and write a comment about why that create table is being used. 
If last_updated is not being used then for now we can remove it. Or are you keeping it for some future use?